online-shop/src/app/guards/auth.guard.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed, async, inject } from '@angular/core/testing';
+
+import { AuthGuard } from './auth.guard';
+
+describe('AuthGuard', () => {
```

Comment:
You should remove the "skeleton" test files if you are not writing real tests there.